releasenotes/notes/0.19/QuantumCircuit.decompose-takes-which-gate-to-decompose-d857da5d0c41fb66.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    :meth:`.QuantumCircuit.decompose` and its corresponding transpiler pass
+    :class:`~qiskit.transpiler.passes.Decompose` now optionally accept a
+    parameter containing a collection of gate names.  If this parameter is given,
+    then only gates with matching names will be decomposed.  This supports
+    Unix-shell-style wildcard matches.  For example::
+
+        qc.decompose(["h", "r[xz]"])
+
+    will decompose any ``h``, ``rx`` or ``rz`` gates, but leave (for example) ``x`` gates untouched.
```

Comment:
Spacing after a code block is big and breaks the sentence into a paragraph:
<img width="655" alt="Screenshot 2021-12-04 at 07 51 14" src="https://user-images.githubusercontent.com/766693/144700686-afa15c65-0b25-46ba-815d-f47d7bc258fa.png">



```suggestion
    Unix-shell-style wildcard matches.  For example, ``qc.decompose(["h", "r[xz]"])`` will decompose any ``h``, ``rx`` or ``rz`` gates, but leave (for example) ``x`` gates untouched.
```

Reply:
This particular one is a CSS problem, not a release note problem, I think - if you look at places where there's a paragraph break with _no_ code, there's no space between paragraphs (but there probably should be), whereas here there's a big space, but it shouldn't be any more than a regular paragraph space.

A quick look shows it's coming from an `article.pytorch-article pre { margin-bottom: 2.5rem; }` CSS rule; I'm not sure we should have that at all, but if it's useful elsewhere, we could also override it with something like a 
```css
#release-notes article.pytorch-article pre {
    margin-bottom: 0;
}
```
because really we only want the paragraph spacing.

Reply:
We definitely want this rule - it's the same thing that causes a weirdly large space in the text-only output from a `jupyter-execute` block.